zulip_bots/zulip_bots/bots/wikipedia/wikipedia.py

```diff
@@ -56,12 +56,15 @@ def get_bot_wiki_response(self, message: dict, bot_handler: ExternalBotHandler)
 
         except requests.exceptions.RequestException:
             logging.error('broken link')
-            return None
+            return 'Uh-Oh ! Sorry ,couldn\'t process the request right now.:slightly_frowning_face:\n' \
```

Comment:
We should probably add test coverage for this.

Reply:
I already have the tests done [here](https://github.com/zulip/python-zulip-api/pull/232/files#diff-418787db0bb7c23c3cf2cf65150abf5eR62)

Reply:
oh, gotcha